### PR TITLE
Update slam_toolbox_rviz_plugin.h

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Fix error:
```
Undefined symbols for architecture x86_64:
  "slam_toolbox::SlamToolboxPlugin::PauseProcessingCb(int)", referenced from:
      slam_toolbox::SlamToolboxPlugin::qt_static_metacall(QObject*, QMetaObject::Call, int, void**) in moc_slam_toolbox_rviz_plugin.cxx.o
ld: symbol(s) not found for architecture x86_64
```

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
I guess this patch will be required in other branches too, I can do that once you approved this change @SteveMacenski

**Review by SteveMacenski — APPROVED** [review]
LGTM, I'd appreciate if you could also do for ROS2 foxy and maybe Melodic if you're up for it.

**Comment by Tobias-Fischer** [comment]
Did eloquent and dashing too - feel free to close these if those branches are not maintained anymore.

**Comment by SteveMacenski** [comment]
Thanks for the contribution and finding that!

**Comment by WilliamWoo45** [comment]
**Required Info:**

- Operating System:
-Ubuntu [16.04]

- Installation steps:
1. cd catkin_ws_slamtoolbox/src
2. git clone -b kinetic-devel https://github.com/SteveMacenski/slam_toolbox.git
3. cd ..
4. catkin_make_isolated
Installed successfully:
[100%] Built target slam_toolbox
<== Finished processing package [1 of 1]: 'slam_toolbox')

- ROS Version
-ROS1, Kinetic

- Version or commit hash:
-Kinetic-devel


Hi! Firstly, thanks for providing such great SLAM Toolbox.
I can run the SLAM process smoothly, but the `SlamToolboxPlugin in Rviz` does not show up as expected. The error is shown as (below figure): "`The class required for this panel, 'slam_toolbox/SlamToolboxPlugin', could not be loaded. Error: According to the loaded plugin descriptions the class slam_toolbox/SlamToolboxPlugin with base class type rviz::Panel does not exist. Declared types are`"

Rviz_plugin_error


I did the modification as @Tobias-Fischer  suggested, commented out the line in [slam_toolbox_rviz_plugin.h](https://github.com/SteveMacenski/slam_toolbox/blob/9b14cf8369a63264fd9935be4cca5cf14386bfc5/rviz_plugin/slam_toolbox_rviz_plugin.h#L77)  and changed the "/" to "::" in [slam_toolbox_default.rviz](https://github.com/SteveMacenski/slam_toolbox/pull/230/commits/f9c9d727db6e68223411fdaedc2c1cebd690548a#diff-54e43590ccbd7ad0c745bb099bfaef173a768e66e617fafcec5bdaf2ebcb4708R32).
I re-compiled successfully. But there is still no `SlamToolboxPlugin` shown in Rviz.

So I'm wondering how could I solve this problem? Or is there any workaround method to **save the serialized map** for latter localization usage? Because I'm mainly trying to implement the localization mode of the SLAM Toolbox (I suppose the `.pgm` and `.yaml` file built by `Hector-SLAM` or `Cartographer` are not usable for the localization mode of SLAM Toolbox).


Thanks a lot! @SteveMacenski 

All the best and regards,
William

**Pull request merged**
